gui/js/measurement-tools-tab.js

```diff
@@ -1,29 +1,52 @@
-init_slider("set_widget_scale", 0.2, 2.0, 0.01, [0.6]);
-
-function add_measurement_tool(name, icon) {
-    const area = $('#measurement-tools');
-    area.append(`<div class='col-4 center' style='padding: 0 5px'>
-                    <label style="width: 100%; height: 100%">
-                        <input id='set_tool_${icon}' type='checkbox' name='measurement-tool' class='radio-button' />
-                        <div class='block btn glass'>
-                            <i style="font-size: 60px" class='material-icons'>${icon}</i> 
-                            <br> 
-                            <span>${name}</span>
-                        </div>
-                    </label>
-                </div>`);
-
-    $('#set_tool_' + icon).change(function () {
-        $('#measurement-tools .radio-button').not('#set_tool_' + icon).prop('checked', false);
-
-        if (this.checked) {
-            window.call_native('set_measurement_tool', name);
-        } else {
-            window.call_native('set_measurement_tool', 'none');
-        }
-    });
+class MeasurementToolApi extends IApi {
+    name = 'measurement_tool';
```

Comment:
Maybe we could improve the naming a bit. The class could be `MeasurementToolsApi `, the name could be `measurementTools`. `addMeasurementTool` could be called simply `add`. Then we could call `callJavascript(
      "CosmoScout.measurementTools.add", "Path", ...);` instead of `callJavascript(
      "CosmoScout.call", "measurement_tool", "addMeasurementTool", "Path", ...);` which increases readability a lot.

Reply:
Done in https://github.com/cosmoscout/csp-measurement-tools/pull/3/commits/8bec1676f441d5a702b754e4b97f68510abce66d